packages/http-client-python/emitter/src/emitter.ts

```diff
@@ -50,9 +50,14 @@ function addDefaultOptions(sdkContext: SdkContext) {
     options["package-mode"] = sdkContext.arm ? "azure-mgmt" : "azure-dataplane";
   }
   if (!options["package-name"]) {
-    options["package-name"] = removeUnderscoresFromNamespace(
-      (sdkContext.sdkPackage.rootNamespace ?? "").toLowerCase(),
-    ).replace(/\./g, "-");
+    const namespace = getRootNamespace(sdkContext as PythonSdkContext<SdkServiceOperation>);
+    const packageName = namespace.replace(/\./g, "-");
+    reportDiagnostic(sdkContext.program, {
```

Comment:
we don't want to raise if there is no `package-name` option. This should be the default going forward: we get the `package-name` from `namespace.replace(...)`. If users pass in a `package-name`, we override the default. So the code should look more like this

```python
const packageName = options["package-name"] || namespace.replace(/\./g, "-")
```

Reply:
ok. then it's fine.